config/routes.rb

```diff
@@ -15,6 +15,17 @@
     resources :listings, except: [:index]
   end
 
+ # post   '/:user_slug/listings'          , 'user/listings#create' , as: :user_listings
+ # get    '/:user_slug/listings/new'      , 'user/listings#new'    , as: :new_user_listing
+ # get    '/:user_slug/listings/:id/edit' , 'user/listings#edit'   , as: :edit_user_listing
+ # get    '/:user_slug/listings/:id'      , 'user/listings#show'   , as: :user_listing
```

Comment:
Leave for the time being.  
They are there to help understand the routes as a glance.